variables.tf

```diff
@@ -110,7 +110,7 @@ variable "docker_machine_spot_price_bid" {
 variable "docker_machine_download_url" {
   description = "(Optional) By default the module will use `docker_machine_version` to download the GitLab mantained version of Docker Machine. Alternative you can set this property to download location of the distribution of for the OS. See also https://docs.gitlab.com/runner/executors/docker_machine.html#install"
```

Comment:
Can you update this description, maybe refer to https://gitlab.com/cki-project/docker-machine and quickly explain why we switch.

Reply:
Fixed the description but didn't mentioned any reason there as I think it is too much information for the variable description. I put some information into the description of this PR.